tide_landing_page.install

```diff
@@ -2456,3 +2457,18 @@ function tide_landing_page_update_8045() {
   $config->set('default_value', ['noImage']);
   $config->save();
 }
+
+/**
+ * Delete Block content campaign with video.
+ */
+function tide_landing_page_update_8046() {
+  $ids = \Drupal::entityQuery('block_content')
+    ->condition('type', 'campaign_with_video')
+    ->execute();
+  if (!empty($ids)) {
+    foreach ($ids as $item) {
+      $block = BlockContent::load($item);
+      $block->delete();
```

Comment:
hey @yeniatencio  . 
I think you are trying to delete the blocks. but you still need to delete its definitions in the hook.

Reply:
Thanks @vincent-gao . I have updated the pr for your review.